cvmfs/catalog_mgr_rw.cc

```diff
@@ -12,9 +12,12 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include <ctime>
```

Comment:
Well, in my workflow, I go through the PR line by line and, among other things, I mark spots that are fixes to the existing code (i.e. I do the analysis, you don't have to do it).  If by the end of the review, we have only two or three lines of pure fixes, we can be sloppy and piggyback on the same PR.  

If it is more than that or if the PR anyway requires careful review (like this one), every line of code we can remove from the core of the patch makes my life easier. It helps me to focus on the important parts and not get distracted by lots of other changes.